packages/eas-cli/src/graphql/queries/AppQuery.ts

```diff
@@ -60,4 +73,64 @@ export const AppQuery = {
     assert(data.app, 'GraphQL: `app` not defined in server response');
     return data.app.byFullName;
   },
+  async byIdWorkflowsAsync(
+    graphqlClient: ExpoGraphqlClient,
+    appId: string
+  ): Promise<Partial<Workflow>[]> {
```

Comment:
I think what you had before was more correct as far as nullability goes. My comment was more to just return the array of workflows to match the function name, but returning just `AppWorkflowsFragment` works, too.

```suggestion
  ): Promise<AppWorkflowsFragment['workflows']> {
```
or
```suggestion
  ): Promise<AppWorkflowsFragment> {
```